libr/anal/p/anal_gb.c

```diff
@@ -1562,7 +1562,7 @@ static int archinfo(RAnal *anal, int q) {
 	case R_ANAL_ARCHINFO_MIN_OP_SIZE:
 		return 1;
 	}
-	return 1;
+	return -1;
```

Comment:
uhm, -1 can result in funky behaviours, isnt? youa re now checking for <=1 so it sohuld be good. but maybe better to give 1 or 0 imho

Reply:
but 1 is valid alignment. -1 indicates unsupported query